Makefile

```diff
@@ -865,6 +865,7 @@ podman-remote-release-%.zip: test/version/version ## Build podman-remote for %=$
 # Downloads pre-built gvproxy and win-sshproxy helpers. See comment on GV_VERSION declaration
 .PHONY: win-gvproxy
 win-gvproxy: test/version/version
+	echo $(GV_VERSION)
```

Comment:
do we need this? I don't mind but then it should likely say GVPROXY_VERSION $(GV_VERSION) so that the output is clear on which version it is.